bindata/manifests/machine-config/files/configure-switchdev.sh.yaml

```diff
@@ -6,18 +6,41 @@ contents:
     #!/bin/bash
     set -eux
     input="/etc/switchdev.conf"
+    UDEV_RULE_FILE='/etc/udev/rules.d/10-persistent-net.rules'
+
     if [ ! -f $input ]; then
       echo "File /etc/switchdev.conf not exist."
       exit
     fi
 
+    append_to_file(){
+      if ! test -f "$2"
+      then
+        echo "$1" > "$2"
+      else
+        if ! grep -Fxq "$1" "$2"
+        then
+          echo "$1" >> "$2"
+        fi
+      fi
+    }
+
+    add_udev_rule_for_sriov_pf(){
```

Comment:
I assume this will keep the PF name as it is?
On openshift, it expects the PF name to be consistent w/ and w/o switchdev mode.

Reply:
Yes, the udev rule is to save the name of the PF